jbpm-container-test/jbpm-container-test-suite/pom.xml

```diff
@@ -0,0 +1,1016 @@
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jbpm</groupId>
+    <artifactId>jbpm-container-test</artifactId>
+    <version>7.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>jbpm-container-test-suite</artifactId>
+
+  <name>jBPM :: Container Test Suite</name>
+
+  <description>jBPM Container Test Suite</description>
+
+  <properties>
+    <surefire.arg.line>
+      -Dfile.encoding=${project.build.sourceEncoding}
+      -Xms256m
+      -Xmx1024m
+    </surefire.arg.line>
+    <wsdl.token>?wsdl</wsdl.token>
+
+    <!-- Datasource properties -->
+    <org.jbpm.persistence.dialect>org.hibernate.dialect.H2Dialect</org.jbpm.persistence.dialect>
+    <org.jbpm.datasource.connection.url>jdbc:h2:mem:test-db</org.jbpm.datasource.connection.url>
+    <org.jbpm.datasource.username>sa</org.jbpm.datasource.username>
+    <org.jbpm.datasource.password></org.jbpm.datasource.password>
+    <org.jbpm.datasource.driver.class>org.h2.Driver</org.jbpm.datasource.driver.class>
+    <org.jbpm.datasource.driver.jdbc.class>org.h2.Driver</org.jbpm.datasource.driver.jdbc.class>
+    <org.jbpm.datasource.hostname/>
+    <org.jbpm.datasource.port/>
+    <org.jbpm.datasource.dbName/>
+    <org.jbpm.persistence.schema/>
+    <org.jbpm.persistence.ds/>
+
+    <!-- Bitronix configuration files for Tomcat 8 -->
+    <bitronix.config.properties>${project.build.testOutputDirectory}/bitronix/btm-config.properties
+    </bitronix.config.properties>
+    <bitronix.resources.properties>${project.build.testOutputDirectory}/bitronix/btm-resources.properties
+    </bitronix.resources.properties>
+
+
+    <version.tomcat8>8.0.33</version.tomcat8>
+    <version.wildfly10>10.0.0.Final</version.wildfly10>
+
+    <!-- WebLogic and WebSphere containers need to be already installed -->
+    <weblogic.home>valid-path-has-to-be-specified-here-or-via-cmd-line</weblogic.home><!-- Example: /home/Servers/wls12130 -->
+    <websphere.home>valid-path-has-to-be-specified-here-or-via-cmd-line</websphere.home><!-- Example: /home/Servers/WebSphere/AppServer -->
+
+    <!-- The EAP 7 binary can't be anonymously downloaded because of the license. It can be downloaded manually
+         and for free (e.g. from http://jbossas.jboss.org/downloads.html) and the zip location needs to be specified
+         here or via system property when running the build (don't forget to use the `file://` prefix when
+         referencing the zip from local filesystem). -->
+    <eap7.download.url>valid-url-for-eap-7-needs-to-be-specified-here-or-via-cmd-line</eap7.download.url>
+
+
+    <cargo.dir>${project.build.directory}/cargo</cargo.dir>
+    <container.log.file>${cargo.dir}/profile-log.txt</container.log.file>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.h2database</groupId>
+        <artifactId>h2</artifactId>
+        <scope>test</scope>
+        <version>1.3.176</version>
```

Comment:
Now it is OK. It was different version before and I had to change it because of WAS.